### ISSUE TITLE
Put support_type into Support table

### ISSUE THREAD

**Issue by danger89** [lbryio/chainquery#128]
Hi,

Its very hard to figure out if a row in the support table is actually just 'support' or really a 'tip'. My proposal would be to introduce `support_type` (I don't care which database type it will be, could be an int or enum or ...).

As long as you can easily distinguish if the support is a **tip** or a **support** (or whatever LBRY could have in the future as well).

Regards,
Melroy van den Berg

**Comment by tzarebczan** [comment]
There's really no clean way to tell right now except for looking at the claim address, which historically may be different and could potentially change (guess we can still track at time of tx). The blockchain team is working on introducing metadata for supports, so we can then mark them appropriately among other cool things we can start to utilize the metadata for (i.e comments, ratings, subscriptions, signed supports)

**Comment by tiger5226** [comment]
Yeah, agreed with @tzarebczan here. There is no really accurate way to do this right now.
